src/classes/frSetupController.cls

```diff
@@ -5,6 +5,24 @@ public with sharing class frSetupController {
 		}
 		private set;
 	}
+	public static String PERSON_ACCOUNT_TYPE {
```

Comment:
This should be removed and the shared variable that defines the type in frDonation, frDonor etc should be used.

Reply:
Removed!